app/scripts/components/analysis/results/timeseries-data.ts

```diff
@@ -35,7 +35,7 @@ export interface TimeseriesDataResult {
   isPeriodic: boolean;
   timeDensity: TimeDensity;
   domain: string[];
-  timeseries: TimeseriesDataUnit[];
+  timeseries?: TimeseriesDataUnit[]; // NOTE: Summaries on collections will not always be available
```

Comment:
To make the type useful, we seem to need a new type. The components like `chart-card` has to have timeseries data, so it doesn't seem to reflect is actual usage to type `timeseires` as an optional. (Also the data with summaries and wo summaries having the same type is confusing, like this line : https://github.com/NASA-IMPACT/veda-ui/pull/786/files#diff-73d60515e81c1007b36e13ea2886f3b4e1457746381934d40af7802e39841022R182) 

How about leaving the `TimeseriesDataResult` as it is, but making `TimeseriesDataWithMissingSummaries` (I just made the variable name up) and use this type explicitly for the data without summaries? 

More food for thought:  when the data is missing summaries, we don't seem to need any attributes from the collection?

Reply:
I like the idea of being explicit but if we dont seem to need any attributes from the collection when summaries is missing from the data, then we can probably just use type `DatasetLayer`. Will make this change